pydantic_ai_slim/pydantic_ai/models/bedrock.py

```diff
@@ -643,20 +643,25 @@ async def _map_user_prompt(part: UserPromptPart, document_count: Iterator[int])
                     else:
                         raise NotImplementedError('Binary content is not supported yet.')
                 elif isinstance(item, ImageUrl | DocumentUrl | VideoUrl):
-                    downloaded_item = await download_item(item, data_format='bytes', type_format='extension')
-                    format = downloaded_item['data_type']
+                    source: dict[str, Any]
+                    if item.url.startswith('s3://'):
+                        source = {'s3Location': {'uri': item.url}}
```

Comment:
There's also a `bucketOwner` field that users may want to set. Maybe we can tell them to encode it as a query param on the URL, and parse it out here?